src/pages/Login.vue

```diff
@@ -275,12 +273,14 @@ export default {
         iconClass: "animate-spin h-4 object-scale-down text-white",
       };
     },
+    /**
+     * whether resendOTPTimer is enabled for resend OTP
+     */
     isResendOTPEnabled() {
-      //return whether resendOTPTimer is enabled for resend OTP
-      if (this.resendOTPTimer == 0 || this.resendOTPTimer == 60) {
-        return false;
+      if (this.resendOTPTimer == 0) {
+        return true;
       }

```

Comment:
single lines don't need a block
```suggestion
      if (this.resendOTPTimer == 0) return true;
```